### ISSUE TITLE
Display group messages in top bar

### ISSUE THREAD

**Issue by janfitz** [Alex-Rose/fb-messenger-cli#55]
For me will be more comfortable if top bar with names of friends could contains also group messages mixed with conversations with single friends. Ideally sorted by delivery time. Because now I can't recieve notification on new group message. What do you think?

**Comment by SamBergeron** [comment]
Yeah I do think having both Groups and personal conversations mixed into 1 is a good feature, for both the header bar and conversation selection.
I just came back from some time away... I'll work on this. :+1:

**Comment by 2E0PGS** [comment]
+1 for this. Makes sense since I only use group convo mostly.

**Comment by jathanasiou** [comment]
@SamBergeron isn't this already implemented? Perhaps it should be closed.

**Comment by janfitz** [comment]
Maybe I have another note regarding this issue. If some group message is used quite often, then it is displayed more times in the top bar.

![snimek z 2018-03-12 09-39-39](https://user-images.githubusercontent.com/12275240/37273365-dc1a0354-25d9-11e8-9067-55bb40a6ce86.png)
